package.json

```diff
@@ -15,15 +15,15 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
+    "deploy": "ember deploy production",
     "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
     "prepare": "./scripts/link-them.sh",
     "start": "ember serve",
     "test": "npm-run-all lint:* test:*",
-    "test:browser": "ember test --test-port=0",
-    "test:ember": "ember test",
-    "test:ember-compatibility": "ember try:each",
+    "test:ember": "ember test --test-port=0",
```

Comment:
I'm not sure why previously `--test-port=0` was needed. Both `ember test --test-port=0` and `ember test` ran fine locally. We may be able to write `ember test` to more closely follow the `package.json` in a new addon.